zerver/lib/test_fixtures.py

```diff
@@ -240,7 +240,10 @@ def template_database_status(database_type: str) -> str:
         # changes, we can safely assume we don't need to run
         # migrations without spending a few 100ms parsing all the
         # Python migration code.
-        paths = glob.glob('*/migrations/*.py')
+        paths = [
+            *glob.glob('*/migrations/*.py'),
+            'requirements/dev.txt',
```

Comment:
Because django-otp has its own migration and we were failing to detect that it needs to be run.